docs/src/main/paradox/client/details.md

```diff
@@ -59,5 +59,25 @@ Scala
 Java
 :  @@snip [GreeterClient.java](/plugin-tester-java/src/main/java/example/myapp/helloworld/LiftedGreeterClient.java) { #with-metadata }
 
+## Rich error model
 
+Beyond status codes you can also use the @link:[Rich error model](https://www.grpc.io/docs/guides/error/#richer-error-model) { open=new }. Currently there is no native support for this concept in akka-grpc. However you can use the following manual approach.
```

Comment:
I think we usually just use 'regular' markdown links:

```suggestion
Beyond status codes you can also use the [Rich error model](https://www.grpc.io/docs/guides/error/#richer-error-model). Currently there is no native support for this concept in akka-grpc. However you can use the following manual approach.
```

Reply:
I think we prefer 'Akka gRPC' in running text, and `akka-grpc` in places like URL's and artifact names:

```suggestion
Beyond status codes you can also use the @link:[Rich error model](https://www.grpc.io/docs/guides/error/#richer-error-model) { open=new }. Currently there is no native support for this concept in Akka gRPC. However you can use the following manual approach.
```